src/main/java/com/hundred/percent/capstone/Invoicify/company/controller/CompanyController.java

```diff
@@ -1,21 +1,38 @@
 package com.hundred.percent.capstone.Invoicify.company.controller;
 
 import com.hundred.percent.capstone.Invoicify.company.dto.CompanyDTO;
+import com.hundred.percent.capstone.Invoicify.company.entity.CompanyEntity;
+import com.hundred.percent.capstone.Invoicify.company.service.CompanyService;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.http.HttpStatus;
 import org.springframework.web.bind.annotation.*;
 
+import java.util.List;
+
 @RestController
 @RequestMapping("/companies")
 public class CompanyController {
 
-    @GetMapping
+    @Autowired
+    CompanyService companyService;
+
+    /*@GetMapping
     public String getCompanies(){
         return "[]";
+    }*/
+
+    @GetMapping
+    public List<CompanyEntity> getCompanies(){
```

Comment:
This should be a company DTO. Company Entity is being used here. Please check

Reply:
We specifically wants to return entity, so response can have ID field.